featuretools/__init__.py

```diff
@@ -8,14 +8,12 @@
 from .primitives import list_primitives
 from .computational_backends.api import *
 from . import tests
-from .utils.pickle_utils import *
 from .utils.time_utils import *
+from .version import __version__
```

Comment:
The instructions on updating the version number in `release.md` should be updated with the new file location